sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataCollector.java

```diff
@@ -96,6 +95,33 @@ synchronized FinalCounters peek() {
         return null;
     }
 
+
+    synchronized void addOtelMetric(TelemetryItem telemetryItem){
+        if (!isEnabled()) {
+            // quick pulse is not enabled or quick pulse data sender is not enabled
+            return;
+        }
+
+        if (!telemetryItem.getInstrumentationKey().equals(instrumentationKeySupplier.get())) {
+            return;
+        }
+
+        Float sampleRate = telemetryItem.getSampleRate();
+        if (sampleRate != null && sampleRate == 0) {
+            // sampleRate should never be zero (how could it be captured if sampling set to zero percent?)
+            return;
+        }
+        int itemCount = sampleRate == null ? 1 : Math.round(100 / sampleRate);
```

Comment:
what is this code being used for?

Reply:
deleted itemCount as it is not being used